addons/crm_iap_lead_website/models/crm_reveal_rule.py

```diff
@@ -113,13 +113,15 @@ def _compute_lead_count(self):
     def action_get_lead_tree_view(self):
         action = self.env["ir.actions.actions"]._for_xml_id("crm.crm_lead_all_leads")
         action['domain'] = [('id', 'in', self.lead_ids.ids), ('type', '=', 'lead')]
-        action['context'] = dict(self._context, create=False)
+        filtered_context = self.with_context(group_by=[])
+        action['context'] = dict(filtered_context._context, create=False)
```

Comment:
As a side note, that's a complicated way to do
```suggestion
        action['context'] = dict(self._context, create=False, group_by=[])
```

Reply:
-1 for this solution, you have the same issue with keys like 'form_view_ref' and probably dozen others.
Why are we passing context from the self instead of context from the action (if any at all)?